week03/assignment03.go

```diff
@@ -0,0 +1,107 @@
+package week03
+
+import (
+	"fmt"
+)
+
+var (
+	ErrRatingNoPlays       = fmt.Errorf("can't review a movie without watching it first")
+	ErrInvalidViewersValue = fmt.Errorf("invalid number of viewers, must be greather than 0")
+	ErrTeatherPlayNoMovies = fmt.Errorf("no movies to play")
+)
```

Comment:
As we'll learn next week these are called sentinel errors and are considered bad practice. Please remove these.